css/settings/_colors.scss

```diff
@@ -0,0 +1,27 @@
+$alto: #dfdfdf;
```

Comment:
Try to reduce the amount of colors used. They seem a lot considering what we've achieved so far. If a color is similar to another, use one of them. In general, you don't want to deal with too many variations.